src/envoy/mixer/http_control.h

```diff
@@ -37,7 +37,8 @@ typedef std::shared_ptr<HttpRequestData> HttpRequestDataPtr;
 class HttpControl final : public Logger::Loggable<Logger::Id::http> {
  public:
   // The constructor.
```

Comment:
ok I will do that.
let me close this PR and I will create a new one.

Thanks.